.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Export environment variables
         run: |
-          echo "SECRET_KEY=$(openssl rand -hex 32)" > .env
```

Comment:
This is used for e2e tests.
For uni tests, we export local `.env` file.
See https://github.com/kernelci/kernelci-api#api-testing

Reply:
I know